### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_Dashboard_ScreenReader: ORCA Screen reader is announcing "Learn More" link as button.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio-mysql#174]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev3; #A11yTCS; #DesktopApp; #GH_MySQLextensionforAzureDataStudio_Linux_Mar23; #MySQL extension for Azure Data Studio; #WCAG4.1.2; #Linux; #ScreenReader; #ORCA;

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "Servers" and select any server from the list and hit ENTER on 'Connect' button.
3. TAB to connected server and right click on it and TAB to "Manage" and hit ENTER key.
4. Start the ORCA screen reader. TAB to "Learn more" link.
5. Observe that ORCA Screen reader is announcing "Learn More" link as button.

### Actual Result:​
ORCA Screen reader is announcing "Learn More" link as button.

### Expected Result:​
ORCA Screen reader should be announcing "Learn More" link as "Learn More link."

### User Impact:
Screen reader users will get confused about the role of the "Learn More", as after selecting link is opening but screen reader announcing it as button.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:

https://user-images.githubusercontent.com/90901519/228772351-cdb54578-2813-475d-8273-495f2a8219a8.MOV

**Comment by nitishsharma25** [comment]
This issue is external to MySQL extension as it is a common UI of Azure Data Studio.

**Comment by PoojaNamde** [comment]
Logged External bug: https://github.com/microsoft/azuredatastudio/issues/22690

As per the [External Bug Process - First Party Dependencies](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDV8MDF8fGM4NTQxMGIxZGFhYjQwMTZmNTIzMDhkYWM5NDgyMWQwfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgwNDM2MDg4Njg1MjIxNDZ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVFkyT0RjMk5EQTROVFkzT1RzeE5qWTROelkwTURnMU5qYzVPekU1T2pjelpqUTNOV00zTFRnNU5Ea3RORFprT1MxaFlURTVMVGN3WlRobU9EWm1ZVGszT0Y4M1lqbG1ZVFV5TmkweVptRXhMVFF5WmpRdFltVXhOeTFsTXpRME5EbGpabVkyTlRSQWRXNXhMbWRpYkM1emNHRmpaWE09fDA5YTBhOTU1NzM0MDQ5NTM1OTJiMDhkYWM5NDgyMWQwfGZlNmZiOTdlMzBhNTRjNzJhNTRhNTU0Y2RjNDg0NzM3&sdata=ZVRCdGg0WldPVlRVQ2RveFFNV1FJUEc0bHc1a2QvWTExbkVLS3V5emQzYz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-ramakatare%40microsoft.com) assigning the bug to the Product Owner. Please assign this bug back to the tester once the "External" issue has been resolved and changes are pushed to the environment that will be available for bug verification.

**Comment by keerthiduvvuri** [comment]
GithubTags:#ResolvedByTT
